Install fails using Mac OS X instructions during make step

# Issue Type: Bug

# stellar-core --version v9.2.0-97-ga3c0cf61
# git rev-parse HEAD a3c0cf6113877243353190a32a206a09291dd80a

# OS: Mac OSX High Sierra, command line install

# Issue description
Using the instructions found at https://github.com/stellar/stellar-core/blob/master/INSTALL.md the install fails during the make step, specifically here:
mkdir -p docs
sed -ne '/^#/,$p' "./docs/software/commands.md" \
            | sed -e '/^%commands%$/{s///; r/dev/stdin' -e '}' \
            ./docs/stellar-core.1.md > tmp.man.md
pandoc -s -f markdown -t man -o "./docs/stellar-core.1.in~" tmp.man.md \
            && mv -f "./docs/stellar-core.1.in~" "./docs/stellar-core.1.in"
/bin/sh: pandoc: command not found
make[2]: [docs/stellar-core.1] Error 127 (ignored)
rm -f tmp.man.md
sed -e "s|%prefix%|/usr/local|g" "./docs/stellar-core.1.in" > "docs/stellar-core.1~" \
            && mv -f "docs/stellar-core.1~" "docs/stellar-core.1"
sed: ./docs/stellar-core.1.in: No such file or directory
make[2]: *** [docs/stellar-core.1] Error 1
make[1]: *** [all-recursive] Error 1
make: *** [all] Error 2

## Actual result
 make errored out due to missing file which itself was a result of missing pandoc executable

## Expected result
Make steps should have succeeded so can proceed with make tests and make install

Note, following up with a PR with the very simple fix for this, which is adding to the install docs as a requirement to do a brew install pandoc

--- a/INSTALL.md
+++ b/INSTALL.md
@@ -52,6 +52,7 @@ When building on OSX, here's some dependencies you'll need:
 - brew install automake
 - brew install pkg-config
 - brew install libpqxx *(If ./configure later complains about libpq missing, try PKG_CONFIG_PATH='/usr/local/lib/pkgconfig')*
+- brew install pandoc
 
 ### Windows
 See [INSTALL-Windows.md](INSTALL-Windows.md)

